### PR TITLE
Get base return (DO NOT MERGE)

### PR DESCRIPTION
**Pull request by faissalMT**

### CODE REVIEW

File: lib/local_authority/use_case/get_base_return.rb [hunk 1]
```diff
@@ -1,15 +1,26 @@
...
 
   def execute(project_id:)
     project = @project_gateway.find_by(id: project_id)
     schema = @return_gateway.find_by(type: project.type)
 
-    { base_return: { id: project_id, data: project.data, schema:schema.schema } }
+    if project.type == 'hif'
+      #Take in some baseline project data and return that hash, unknowns can be nil
+
+      return_template_populator = @populate_return_template
```
Comment (danielburnley) [inline lib/local_authority/use_case/get_base_return.rb]: I wouldn't worry about assigning it to a temp variable there, makes it slightly less readable IMO

File: lib/local_authority/use_case/populate_return_template.rb [hunk 2]
```diff
@@ -4,8 +4,13 @@ def initialize(template_gateway:)
   end
 
   def execute(type:, data:)
-    template = @template_gateway.find_by(type: type)
-    populated_data = populate_hash(template.layout.dup, data)
+    populated_data = @template_gateway.find_by(type: 'hif').layout
+
+    populated_data[:infrastructure][0][:targetSubmission] = data[:infrastructure][0][:submissionEstimated]
+    populated_data[:infrastructure][0][:targetGranted] = data[:infrastructure][0][:grantEstimated]
+    populated_data[:infrastructure][0][:submissionEstimated] = data[:infrastructure][0][:submissionEstimated]
```
Comment (danielburnley) [inline lib/local_authority/use_case/populate_return_template.rb]: I don't _think_ estimated will be in the baseline data from memory

File: spec/unit/local_authority/use_case/populate_return_template_spec.rb [hunk 3]
```diff
@@ -1,243 +1,56 @@
...
+        ]
+      }
     end
   end
+  let(:project_type) {'hif'}
+  let(:baseline) do
+    {
+      infrastructure: [
+        {
+          submissionEstimated: 'baseline data to be copied',
```
Comment (danielburnley) [inline spec/unit/local_authority/use_case/populate_return_template_spec.rb]: I think this will be the `target` values

**Comment by faissalMT** [comment]
Obsoleted by #34

**Pull request closed without merging**